addons/mail/static/src/components/message/message.xml

```diff
@@ -37,9 +37,11 @@
                         </div>
                     </t>
                     <t t-else="">
-                        <div class="o_Message_date o_Message_sidebarItem o-message-squashed" t-att-class="{ 'o-message-selected': isSelected }">
-                            <t t-esc="shortTime"/>
-                        </div>
+                        <t t-if="message.date">
+                            <div class="o_Message_date o_Message_sidebarItem o-message-squashed" t-att-class="{ 'o-message-selected': isSelected }">
```

Comment:
Why not put the `t-if="message.date"` directly on the tag `div` to minimize the diff?

Reply:
This is a guideline we use for Discuss.
Basically, the idea is that it's easier to read/catch the `t-if` on it's own tag rather than on a never ending `div` tag.